src/layout/ActionButton/ActionButtonComponent.tsx

```diff
@@ -1,18 +1,14 @@
 import React from 'react';
 
-import { Button } from '@digdir/designsystemet-react';
-
 import type { PropsFromGenericComponent } from '..';
 
+import { Button, type ButtonColor, type ButtonVariant } from 'src/app-components/button/Button';
 import { useProcessNavigation } from 'src/features/instance/ProcessNavigationContext';
 import { Lang } from 'src/features/language/Lang';
-import { ButtonLoader } from 'src/layout/Button/ButtonLoader';
```

Comment:
I see the `ButtonLoader` component is no longer in use now, so it should probably be deleted. :wastebasket: